### PR TITLE
feat: sentry capture with standard context

### PR DESCRIPTION
**Pull request by MatsuoTakuro**
- we are only supporting the sentry capture with `echo.Context` now.

### CODE REVIEW

File: internal/sentry/capture.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+)
+
+func CaptureException(ctx context.Context, err error,
+	logger echo.Logger, sentryOn bool, noHubMsg string,
+) {
+	if err == nil {
+		return
+	}
+
+	if !sentryOn {
```
Comment (zhongdalu-trechina) [inline internal/sentry/capture.go:19]: Here, the error log is only output when `sentryOn` is false. I want to print the error log regardless of whether sentry is on or off. What do you think?

Comment (MatsuoTakuro) [reply internal/sentry/capture.go:19]: Hmm, I guess users don't necessarily expect the capture funcs to call `echo.Logger.Error()` or `Info()`.
In my projects  `Errorj()` or `Infoj()` is usually used for structured log output.

Comment (zhongdalu-trechina) [reply internal/sentry/capture.go:19]: you mean in the project we can do it like this?
```
bean. Errorj(...)
sentry.CaptureException(...)
```
but if you set `sentryOn` off, the error log will print twice in the above case.

Comment (zhongdalu-trechina) [reply internal/sentry/capture.go:19]: i.c. It's ok for me.
but can you wrap some function contains both `logger.error` and  `sentry.CaptureException`? 
In my opinion, `sentry.CaptureException` may be lost in some cases, but `logger.error` rarely happens, so I want to record the error in two ways ->  `logger.error` and  `sentry.CaptureException`

Comment (MatsuoTakuro) [reply internal/sentry/capture.go:19]: OK, what about if we provide another wrap func like `captureAndLogException` ?

Comment (zhongdalu-trechina) [reply internal/sentry/capture.go:19]: perfect for me

Comment (MatsuoTakuro) [reply internal/sentry/capture.go:19]: Nice, let me create it.

**Comment by MatsuoTakuro** [comment]
[refactor: 💡 change pacakge structure after feedback](https://github.com/retail-ai-inc/bean/pull/236/commits/6f9d4da3d5fea334766e03e626b5c23785ce6f79) 
[6f9d4da](https://github.com/retail-ai-inc/bean/pull/236/commits/6f9d4da3d5fea334766e03e626b5c23785ce6f79)
- Move config from top level `bean` to new `config` package
- Move logger from top level `bean` to new `log` package
- Move sentry related funcs from top level `bean` to `trace` package
- Add `WithEcho` suffix to existing sentry capture func names cuz they take `echo.Context`.
- Create new funcs that log and capture error/message with sentry in `trace` package.

File: trace/capture.go
Comment (zhongdalu-trechina) [inline trace/capture.go:1]: In my personal opinion, there is a little bit repetitive code in this file. can we try to reuse it?

Comment (MatsuoTakuro) [reply trace/capture.go:1]: ok, let me change it to a bit more reused version.

Comment (MatsuoTakuro) [reply trace/capture.go:1]: Changed

**Review by zhongdalu-trechina — APPROVED** [review]

**Pull request merged**